### DISCUSSION TITLE
Angular DevTools support

### DISCUSSION THREAD

**Discussion by ceceliacreates** [RecordReplay/devtools#6081] [Ideas]
User feedback:

are there any plans for any sort of enhancement / integration with Angular and NgRx / Redux at all?

**Comment by markerikson** [comment]
Good news - we're specifically planning to add support for the Redux DevTools shortly!  I plan to start working on implementing that in the next couple weeks.  (And since there's a bunch of other libraries that can make use of the Redux DevTools, like Zustand, Jotai, and NgRx, they _should_ all benefit from that.)

We also absolutely want to integrate and support devtools for frameworks like Angular, Vue, Ember, and Svelte as well.  I don't think we have those specifically tied to the roadmap atm, but very much something we would like to add.
